compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -1509,6 +1509,13 @@ impl<'v> RootCollector<'_, 'v> {
                         }
                         _ => unreachable!(),
                     };
+                    let Ok(instance) = self.tcx.try_normalize_erasing_regions(
```

Comment:
this code is unnecessarily defensive but i'd rather not turn this into an ICE with an impossible-to-normalize opaque type or something.